### PR TITLE
[Logs] New Relic Analytics Integration.

### PR DESCRIPTION
**Pull request by angelampcosta**

### CODE REVIEW

File: content/fundamentals/data-products/analytics-integrations/new-relic/index.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+## Task 1 - Install the Cloudflare Network Logs quickstart
+
+1.  Log in to New Relic.
+2.  Click the Instant Observability button (top right).
+3.  Search for **Cloudflare Network Logs**.
+
+![Cloudflare Network Logs install screen](/fundamentals/static/images/new-relic/screenshots/cloudflare-network-logs.png)
+
+4.  Click **Install this quickstart**.
+5.  Follow the steps to deploy.
```
Comment (deadlypants1973) [inline content/fundamentals/data-products/analytics-integrations/new-relic/index.md:28]: Are these steps in the New Relic dashboard? Or in our docs? If docs, maybe link?

File: content/fundamentals/data-products/analytics-integrations/new-relic/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+### Overview
+
+Get a quick overview of the most important metrics from your websites and applications on the Cloudflare network.
+
+![Cloudflare Network Logs install screen](/fundamentals/static/images/new-relic/dashboard/dash-1.png)
+
+### Security
+
+Get insights on threats to your websites and applications, including number of threats taken action on by the WAF, threats over time, top threat countries, and more.
```
Comment (deadlypants1973) [inline content/fundamentals/data-products/analytics-integrations/new-relic/index.md]:
```suggestion
Get insights on threats to your websites and applications, including number of threats taken action on by the Web Application Firewall (WAF), threats over time, top threat countries, and more.
```

File: content/fundamentals/data-products/analytics-integrations/new-relic/index.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+### Performance
+
+Identify and address performance issues and caching misconfigurations. Metrics include total requests, total versus cached requests, total versus origin requests.
+
+![Cloudflare Network Logs performance metrics screen](/fundamentals/static/images/new-relic/dashboard/dash-3.png)
+
+### Reliability
+
+Get insights on the availability of your websites and Applications. Metrics include, edge response status over time, percentage of 3xx/4xx/5xx errors over time, and more.
```
Comment (deadlypants1973) [inline content/fundamentals/data-products/analytics-integrations/new-relic/index.md]:
```suggestion
Get insights on the availability of your websites and Applications. Metrics include, edge response status over time, percentage of `3xx`/`4xx`/`5xx` errors over time, and more.
```

File: content/fundamentals/data-products/analytics-integrations/new-relic/index.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+3.  Search for **Cloudflare Network Logs**.
+
+![Cloudflare Network Logs install screen](/fundamentals/static/images/new-relic/screenshots/cloudflare-network-logs.png)
+
+4.  Click **Install this quickstart**.
+5.  Follow the steps to deploy.
+
+## Task 2 - View the Cloudflare Dashboards
+
+You can view your dashboards on the New Relic dashboard page. They include:
```
Comment (deadlypants1973) [inline content/fundamentals/data-products/analytics-integrations/new-relic/index.md]: Is 'they' here the different dashboard? Kind of confused how it links to the rest of the sections below.

**Review by deadlypants1973 — APPROVED** [review]

File: content/fundamentals/data-products/analytics-integrations/new-relic/index.md (hunk 1 above)
Comment (tanushree-sharma) [reply content/fundamentals/data-products/analytics-integrations/new-relic/index.md:28]: steps are in the New Relic dashboard

File: content/fundamentals/data-products/analytics-integrations/new-relic/index.md (hunk 4 above)
Comment (tanushree-sharma) [reply content/fundamentals/data-products/analytics-integrations/new-relic/index.md]: maybe just remove "They include:"

**Review by tanushree-sharma — APPROVED** [review]

**Pull request merged**